feat: verbatimModuleSyntax

farreldarian opened the pull request.

jansedlon commented on packages/generator/src/lib/syntaxes/imports.ts:
```diff
@@ -1,24 +1,36 @@
 		render() {
-			return `import { ${names.join(', ')} } from '${renderImportPath(path)}';`
+			// biome-ignore format: off
+			return `import ${isTypeImport ? 'type' : ''} { ${names.join( ', ')} } from '${renderImportPath(path)}';`
```
```suggestion
			return `import ${isTypeImport ? 'type ' : ''}{ ${names.join( ', ')} } from '${renderImportPath(path)}';`
```

jansedlon commented on packages/generator/src/lib/syntaxes/imports.ts:
```diff
@@ -1,24 +1,36 @@
 		render() {
-			return `import ${name} from '${renderImportPath(path)}';`
+			// biome-ignore format: off
+			return `import ${isTypeImport ? 'type' : ''} ${name} from '${renderImportPath(path)}';`
```
```suggestion
			return `import ${isTypeImport ? 'type ' : ''}${name} from '${renderImportPath(path)}';`
```

jansedlon commented:
Just the two small changed, otherwise LGTM. Without the changes, there are 2 spaces.
Eg.

```ts
import  { pgEnum } from 'drizzle-orm/pg-core';
//         ||
```

farreldarian commented:
> Btw i added the `docker-compose.yaml` purposefully so that any other person who would like to contribute had easier time to setup dev environment.

Added
